share/spice/pubmed/pubmed.js

```diff
@@ -0,0 +1,97 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_pubmed = function(api_result){
+        if (!api_result || api_result.error) {
+            return Spice.failed('pubmed');
+        }
+        api_result = api_result.esearchresult.idlist;
+        var url = 'http://eutils.ncbi.nlm.nih.gov/entrez/eutils/esummary.fcgi?db=pubmed&retmode=json&rettype=abstract&retmax=10&id=';
+        $.ajax({
+            url: url + api_result.join(),
+            async: false,
+            dataType: 'json',
+            success: function(r) {                                                                                             
+                delete r.result.uids;
+                api_result = r.result;
+                var articles = Object.keys(api_result).map(function(key){return api_result[key]});
+                api_result = articles;
+            }
+        });    
+        Spice.add({
+            id: "pubmed",
+            name: "Pubmed",
+            data: api_result,
+            meta: {
+                sourceName: "Pubmed",
+                sourceUrl: 'http://www.ncbi.nlm.nih.gov/pubmed/',
+                rerender: [ 'description' ]
+            },
+            normalize: function(item) {
+                var boxData = [{heading: 'Article'}];
```

Comment:
You don't need this :)

It's only if you want to create an InfoBox like we use in Drinks, for example: https://duckduckgo.com/?q=pokemon+bulbasaur&ia=pokedex